### ISSUE TITLE
Latest UE4 version

### ISSUE THREAD

**Issue by MaitreDede** [AndrewScheidecker/BrickGame#11]
Hi,

What about porting BrickGrid to later versions of the UE4 engine ? :)

**Comment by Ryvar** [comment]
Couple notes if you want to do it yourself, since I have been for a while and maybe other people will benefit:
The Engine branch changes needed for 4.7.6 are virtually identical to Andrew's last checkin to his 4.5 branch. Just leave out the "DrawDynamicElements" changes in StaticMeshRender.cpp (DrawDynamicElements has been removed in favor of GetDynamicMeshElements). Note that I don't use the manual AO approach of BrickGrid for my own voxel work - I'm just in it for the directional culling - so there may be additional work needed for DeferredShadingCommon.usf (last I tried duplicating those changes was 4.6 and I'm pretty sure it broke the build).

All told takes about 15 minutes to patch in updated directional culling, in my experience.

For the plugin/project-side code it shouldn't require more than running it through Rama's 4.6 and 4.7 transition guides:
https://forums.unrealengine.com/showthread.php?53389-4-6-Transition-Guide
https://forums.unrealengine.com/showthread.php?59962-4-7-C-Transition-Guide

Haven't attempted it since 4.6 release but I remember it being super-straightforward.
